src/compile/selection/interval.ts

```diff
@@ -94,6 +110,31 @@ const interval: SelectionCompiler<'interval'> = {
         }
       ]
     });
+
+    // Cursor signal
+    const cursorEvents = [
+      ...(selCmpt.viewCursor
+        ? [
+            {events: 'scope:mouseover', update: "'crosshair'"},
+            {events: 'scope:mouseout', update: JSON.stringify(selCmpt.viewCursor)}
+          ]
+        : []),
+      ...(selCmpt.mark.cursor
+        ? [
+            {events: `@${selCmpt.name + BRUSH}:mouseover`, update: JSON.stringify(selCmpt.mark.cursor)},
```

Comment:
This will use `"` while the code below uses `'` quotes. Why don't we just use `value` or a string template?